### PR TITLE
Update ExecuteTime.js for IPython 3.x

### PR DESCRIPTION
**Pull request by lucasb-eyer**
I had to make these changes in order to get this extension to work for me. I'm using a very recent 3.x version of IPython. Please review carefully, as this is the first time I am touching any IPynb extension code!

I hope the `require.toUrl` part will work for everyone, because I did install this extension manually.

### CODE REVIEW

File: usability/execute_time/ExecuteTime.js [hunk 1]
```diff
@@ -6,8 +6,7 @@
 //
 // Execute timings - display when a cell has been executed lastly and how long it took
 // A double click on the box makes it disappear
-
-(function (IPython) {
+define(["require", "jquery", "base/js/namespace"], function (require, $, IPython) {
```
Comment (juhasch) [inline usability/execute_time/ExecuteTime.js]: Can you add `"base/js/events",` and `events` in the define call ?

File: usability/execute_time/ExecuteTime.js [hunk 2]
```diff
@@ -170,10 +169,10 @@
     patchCodecellExecute();
 
     $([IPython.events]).on('ExecuteCell.ExecuteTime',executionStartTime);
-    $([IPython.events]).on('status_idle.Kernel', executionEndTime);
+    $([IPython.events]).on('kernel_idle.Kernel', executionEndTime);
```
Comment (juhasch) [inline usability/execute_time/ExecuteTime.js]: Can you replace the references to `$([IPython.events])` with `events` ?

**Comment by lucasb-eyer** [comment]
Amended your suggestions and tested them.

**Comment by juhasch** [comment]
LGTM. Thanks.

**Pull request merged**